replica_set.go

```diff
@@ -216,6 +216,7 @@ func (r *ReplicaSet) Stop() error {
 }
 
 func (r *ReplicaSet) stop(hard bool) error {
+	r.Stats.BumpSum("replica.stop", 1)
```

Comment:
No need, since every metric sent, includes replica name as a tag.
Datadog is great :)

On Mon, Nov 16, 2015 at 1:07 PM, Alexandre Gerlic <notifications@github.com>
wrote:

> In replica_set.go
> <https://github.com/intercom/dvara/pull/17#discussion_r44920686>:
>
> > @@ -216,6 +216,7 @@ func (r *ReplicaSet) Stop() error {
> >  }
> >
> >  func (r *ReplicaSet) stop(hard bool) error {
> > +	r.Stats.BumpSum("replica.stop", 1)
>
> don't need to add the replica's name as tag ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/intercom/dvara/pull/17/files#r44920686>.
>